workflows/differentialabundance.nf

```diff
@@ -90,26 +179,26 @@ workflow DIFFERENTIALABUNDANCE {
     //
     // MODULE: MultiQC
     //
-    workflow_summary    = WorkflowDifferentialabundance.paramsSummaryMultiqc(workflow, summary_params)
-    ch_workflow_summary = Channel.value(workflow_summary)
-
-    methods_description    = WorkflowDifferentialabundance.methodsDescriptionText(workflow, ch_multiqc_custom_methods_description)
-    ch_methods_description = Channel.value(methods_description)
-
-    ch_multiqc_files = Channel.empty()
-    ch_multiqc_files = ch_multiqc_files.mix(ch_workflow_summary.collectFile(name: 'workflow_summary_mqc.yaml'))
-    ch_multiqc_files = ch_multiqc_files.mix(ch_methods_description.collectFile(name: 'methods_description_mqc.yaml'))
-    ch_multiqc_files = ch_multiqc_files.mix(CUSTOM_DUMPSOFTWAREVERSIONS.out.mqc_yml.collect())
-    ch_multiqc_files = ch_multiqc_files.mix(FASTQC.out.zip.collect{it[1]}.ifEmpty([]))
-
-    MULTIQC (
-        ch_multiqc_files.collect(),
-        ch_multiqc_config.collect().ifEmpty([]),
-        ch_multiqc_custom_config.collect().ifEmpty([]),
-        ch_multiqc_logo.collect().ifEmpty([])
-    )
-    multiqc_report = MULTIQC.out.report.toList()
-    ch_versions    = ch_versions.mix(MULTIQC.out.versions)
+//    workflow_summary    = WorkflowDifferentialabundance.paramsSummaryMultiqc(workflow, summary_params)
```

Comment:
Unless you have immediate plans to add MultiQC in a follow-up PR, I'd suggest removing the module completely for now

Reply:
Good call, I haven't got that far yet - I'll add a ticket for the MultiQC - thanks for the review